src/polkadot/polkadotEngine.ts

```diff
@@ -190,8 +185,7 @@ export class PolkadotEngine extends CurrencyEngine<PolkadotTools> {
       } catch (e: any) {
         if (
           typeof e?.message === 'string' &&
-          // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-          e.message.includes('Subscan /scan/transfers failed with 429')
+          e.message.includes('Subscan /scan/transfers failed with 429') === true
```

Comment:
We are getting an error because `e` is `any`, and so ESLint doesn't know what `e.message.includes` returns. Another fix would be to do `e instanceof error`, instead of `e?.message === 'string'` on the line above. That would tell ESLint that this is an error, and so it would realize the type of `message.includes`

Reply:
Nice, done.